packages/aws-cdk-lib/aws-apigatewayv2/lib/http/integration.ts

```diff
@@ -326,6 +345,7 @@ export abstract class HttpRouteIntegration {
         secureServerName: config.secureServerName,
         parameterMapping: config.parameterMapping,
         credentials: config.credentials,
+        timeout: config.timeout,
```

Comment:
This ended up being a "feature" PR as well, the HTTP integration for API Gateway V2 did not implement the `timeout` prop

Reply:
@nmussy the apigatewayv2 is for http and web sockets and the limit is not adjustable for those yet. Only for the apigateway

https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html

Revert the changes from v2

Reply:
This change is only adding the ability to set the `timeout` prop for the API Gateway V2 HTTP integrations. I'd already added this to the WebSocket integrations previously, see #29566

Reply:
@nmussy  thanks for making the change